src/charm.py

```diff
@@ -53,25 +61,14 @@ def __init__(self, *args):
             self._on_provide_certificate_action,
         )
 
-    def _on_config_changed(self, event: ConfigChangedEvent) -> None:
-        """Triggered when the Juju config is changed.
+    def _on_install(self, event: ConfigChangedEvent) -> None:
+        """Handles the install event.
 
-        The charm is will be in Active Status and ready to handle actions.
+        The charm will be in Active Status and ready to handle actions.
 
         Args:
             event (ConfigChangedEvent): Juju event.
```

Comment:
Same thing in the docstring.
```suggestion
            event (InstallEvent): Juju event.
```

Reply:
✅